db/connect.js

```diff
@@ -1,12 +1,5 @@
-const mongoose = require('mongoose')
+const mongoose = require("mongoose");
 
-const connectDB = (url) => {
-  return mongoose.connect(url, {
-    useNewUrlParser: true,
-    useCreateIndex: true,
-    useFindAndModify: false,
-    useUnifiedTopology: true,
-  })
-}
+const connectDB = (url) => mongoose.connect(url);
```

Comment:
You have done this in very simplistic way but often you may have to manage the bugs and waring with options like the instructors did in the video.

Reply:
Thank you for noticing and point that out. The options in the instructor's boilerplate code for `db/connect.js` would be advisable if I have adopted the instructor's use of Mongoose v. 5.13.2, which came with his boilerplate `package.json`. As I've opted to use Mongoose v. 6.4.6, under which the options are [deprecated](https://mongoosejs.com/docs/migrating_to_6.html) (see "No More Deprecation Warning Options" in bullet list), the options are now by default all set to the specs as spelled out in the instructor's boilerplate, which has become redundant.